pom.xml

```diff
@@ -61,6 +61,12 @@
             <artifactId>jetty-servlets</artifactId>
             <version>9.4.18.v20190429</version>
         </dependency>
```

Comment:
Why did you add this dependency? 

CC : - @bhavya121999

Reply:
To add the CrossOriginFilter class to the Dropwizard environment..we need to add a dependency on Jetty’s “servlets” module .